_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -0,0 +1,104 @@
+---
+id: akn7bn
+name: iframe with negative tabindex has no interactive elements
+rule_type: atomic
+description: |
+  This rule checks that `iframe` elements with a negative `tabindex` attribute value contain no interactive elements.
+accessibility_requirements:
+  wcag20:2.1.1: # Keyboard (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G202: # Ensuring keyboard control for all functionality
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any `iframe` element that has a negative number as a `tabindex` [attribute value][].
+
+## Expectation
+
+None of the test target's [nested browsing context][] includes elements that are [visible][] and part of the [sequential focus navigation][]. An element is "included" in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
```

Comment:
As far as I understand, an `iframe` only has one nested browsing context, not several (so "none of" is not needed).

Reply:
Fair point, I'll fix that.